processing/src/main/java/io/druid/segment/DimensionHandlerUtils.java

```diff
@@ -263,4 +264,24 @@ public static Float convertObjectToFloat(Object valObj)
       throw new ParseException("Unknown type[%s]", valObj.getClass());
     }
   }
+
+  public static Long getIntegralFromFloatString(String floatStr)
```

Comment:
Please add a Javadoc to this method noting that it returns non-null Long only if the string is exactly integral (like 42.0)